src/pages/BanquetFacility/BanquetFacility.scss

```diff
@@ -1,55 +1,15 @@
-.banquet-facility{
-
+.banquet-facility {
+  display: grid;
+  margin-top: 24px;
   margin-bottom: 15rem;
+}
 
-  &__body{
-    margin-bottom: 4.5rem;
-  }
-
-   &__triple{
-    display: grid;
-    grid-template-columns: repeat(3, 1fr);
-    grid-gap: 24px;
-    margin-top: 24px;
-  }
-
-  &__image{
-    grid-column-start: 1;
-    grid-column-end: 2;
-    width: 100%;
-
-    &--triple{
-      object-fit: cover;
-      width: 100%;
-    }
-  }
-  
-  &__section-heading{
-    margin-bottom: 44px;
-  }
-
-  &__section{
-    grid-column-start: 2;
-    grid-column-end: 3;
-  }
-
-  &__container{
-    display: grid;
-    grid-template-columns: 5fr 4fr;
-    gap: 28px;
-
-    @media screen and (max-width: $breakpoint-tablet){
-      display: block;
-    }
-  }
-
-  &__body--right{
-    padding-bottom: 2.5rem;
-    
-    @media screen and (min-width: $breakpoint-tablet){
-      h4{
-        margin-top: 0px;
-      }
+@media screen and (max-width: $breakpoint-tablet) {
+  .banquet-facility {
+    margin-bottom: 4rem;
+     
+    h4 {
```

Comment:
this h4 styling should be deleted no? this should be in a different component now i believe